### DISCUSSION TITLE
Who's Hiring (Feb 2024)

### DISCUSSION THREAD

**Discussion by manovotny** [vercel/next.js#61896] [Show and tell]
# What is this?

This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

## Companies

Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

### Sample Job Post

Here's a good example of a job post.

> Vercel | Software Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers#positions)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our software engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please [email me](mailto:michael.novotny@vercel.com) and let me know so I can send you some swag. 👍 

If you're a developer looking to get hired, please use #61897 instead.

Several companies have [hired someone](https://twitter.com/nutlope/status/1610667757565083648) as a result!

**Comment by mwickett** [comment]
HashiCorp | Senior Web Engineer | Remote (US, Canada) | Full-time | $136 - 172k base | [Link](https://www.hashicorp.com/career/5421095)

HashiCorp's Web Presence team is hiring for two senior engineering roles. We work on [hashicorp.com](https://www.hashicorp.com) and [developer.hashicorp.com](https://developer.hashicorp.com) with a focus on enabling practitioners to be successful with HashiCorp products, and helping drive growth for the company. 

I've been here for 5+ years and have continued to grow, be challenged and have new opportunities to create impact. We've got a solid team who really believe in working in the open together, making pragmatic choices and supporting each other. Happy to chat about the roles anytime!

**Reply by ibarczewski** [reply]
Best way to reach you?

**Comment by chris-stytch** [comment]
Stytch | Frontend Engineer | San Francisco | Full-time | $155-180k base | [Link](https://jobs.ashbyhq.com/stytch/216e38c6-5b9c-4e2b-82c1-9e85da5af1a1)

Stytch creates products that make building authentication a delightful experience, both for developers and for end users. We’re looking for a frontend engineer to join our Web team. As a frontend engineer on our team, you’ll partner with product, design, and other engineers to design and build features that empower our customers to build seamless authentication flows.

I've been here at Stytch for over two years now and its been a blast to build with our amazing team. Feel free to reach out to me with any questions! Links in bio.

**Reply by ibarczewski** [reply]
Hey, I went to your bio, but there doesn't seem to be any links that actually let me message you (I think your profiles might have that disabled). Any other ways to reach you?

**Reply by chris-stytch** [reply]
Just sent you an email!

**Comment by ThePerryR** [comment]
FTL Finance | Junior Software Engineer | St Charles, MO (Remote is possible) | Full-time | $75-95k base + up to 10% bonus | [Link](https://lewis-and-clark-partners2.prismhr-hire.com/job/747949/junior-software-engineer)

FTL Finance is lending company that focuses on financing HVAC repairs. We also have a SaaS platform that we've built that other lenders can use. The company is a little over 50 people, but the software engineering team is just me and one other person. We work with Next.js, Typescript, Tailwind, and MySQL.

The company has been maturing it's technology over the past several years, and it's been really exciting to be a part of shaping that. And if you join our team you will also have the ability to make a large impact on the company, and learn a lot. If you love building web apps, please checkout the link!

Here's a overview of the SaaS platform we're building https://www.apptracker.dev/

**Reply by newaritra** [reply]
Hi,hope you're doing well. This seems like a great opportunity to learn and engineer! Please reach out to me if my CV appeals to you:
[Resume](https://drive.google.com/file/d/1rQ_fMKav-IZH1Uu59aCo72BMH5dLGrKP/view?usp=drivesdk)

**Reply by apophis51** [reply]
# Hi
 @ThePerryR  Your 2k gitcommits beats my 1.4k commits 😁. I am Working on my application for your position at the moment. Lets see how I do 🥲

## About Me

I have My portfolio website at https://malcmind.com and its is build on next.js by hand with tailwindcss, daysiUi

It works best online with chrome, although i'm trying to optimize for mobile and other browsers as well

## Some Note Worthy Apps to look at there: 

### My Cybersecurity Blogs - > https://malcmind.com/ProgrammingBlogs/ffuf---hacker-tools 
- Its a front-end application on NextJs I'm still developing that is meant to teach cyber techniques
- The MVP works "more-or-less" but I have a ton of content to write and features to launch
- It has a full blown integrated terminal that can be used to "run the lessons"  built with Xterm and Websockets (library 'ws'), and the app api is deployed on a customed configured Docker Container that is running on express.js

### My Crypto Game -> https://malcmind.com/Crypto/Crypto-Games-and-Predictions 
- my front-end for it is probably a bit childish... but it's meant to showcase my skill on integrating Nextjs with anything - in this case with Ethereum smart contracts.
- this nextjs application can interface with my solidity smart contract to play the "Guess the number game" where if you guess the number you win the jackpot. 

### My Crypto Prediction System -> https://malcmind.com/Crypto/Crypto-Predictions
This is an awseome NextJs app built to predict the next good crypto to buy 
- unfortunately my server requires manually restarting for daily algo updates... but I'm working to fix this
- I made the AI on brain.js but considering switching over to tensorflow.  Its using a Neural net to determine the likely price in 10 day and 1 day terms... so far it seems to be promissing, but fails to predict high upswings.. work in progress!
- so far it will determine the best crypto to buy with AI trained on trading volume, and trading price for Polkadot, Bitcoin, Ethereum, Solana, and Cardno

### I code daily :)
- add me on linkdin! https://www.linkedin.com/in/malcolm-vernon/ :)

**Comment by clodal** [comment]
One X Tech | Senior Software Engineer (React/Next/Tailwind) | Remote (Global) | Full-time | Competitive Salary | [Link](https://www.onextech.com/careers)

One X Tech is actively seeking a Senior Software Engineer specializing in React and TailwindCSS for a fully remote position. Our development team is global, focusing on modern web applications using NextJS, Vercel, Supabase, Typescript, and TailwindCSS. We prioritize candidates who are comfortable with the GMT+8 timezone and have solid experience in Javascript, particularly with React (NextJS) and TailwindCSS.

With a culture of innovation, One X Tech serves leading enterprises by building technological advancements. This role offers the chance to join an energetic and dynamic team dedicated to excellence in modern web application development. If you're passionate about pushing the boundaries of web development and have the skills listed, we would love to hear from you. Apply through the link provided or send your resume directly to careers@onextech.com with **"[Next Feb 2024]: Remote Software Engineer Application"** as the subject line. Join us in creating extraordinary digital experiences.
